js/ai/src/prompt.ts

```diff
@@ -102,27 +98,7 @@ export interface ExecutablePrompt<
    */
   stream<Out extends O>(
```

Comment:
Also why do we have `Out extends O` here?

Reply:
I swear there was a reason... but don't remember... some weird inference issue. Will remove for now....

Reply:
yes, remembered the reason -- it's to allow overriding it at the call site... ex:
```ts
await ai.prompt('recipe')<typeof RecipeSchema>(input)
```